### PR TITLE
Feature/chalkboard

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
Please check the type of change your PR introduces:
- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Doesn't render question, lives, category, question number, or answers on the chalkboard component
- Question, answers, lives components not written
- Api pulls weird bug dealing with quotes from api

## What is the new behavior?

- Renders question, lives, category, question number, or answers on the chalkboard component
- Question, answers, lives components are written
- Bug fixed

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -6,33 +6,28 @@ import Chalkboard from '../Chalkboard/Chalkboard'
...
-          <section className='body'>
-            <div className='chalkboard'>Chalkboard</div>
-          </section>
-        </div>
-      </main>
+      <p>{props.error}</p>
     )
   }
-  
+  if(props.isLoading) {
```
Comment (lilianaweimer) [inline src/App/App.js:18]: Nice conditional rendering!

File: src/Chalkboard/Chalkboard.js [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+          <section className='body'>
+            <section className='chalkboard'>
+            <Question question={question.question} />
+            <Answers question={question} />
+            </section>
+          </section>
+        </div>
+      </main>
+    )
+  }
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.js:26]: nice job moving this from app.

File: src/Answers/Answers.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+const Answers = (props) => {
+  const correctAnswer = props.question.correct_answer
+  const incorrectAnswers = props.question.incorrect_answers
+  const allAnswers = [...incorrectAnswers, correctAnswer]
```
Comment (lilianaweimer) [inline src/Answers/Answers.js:7]: Putting all the answers in one array for displaying them is a great idea! Glad we did it.

File: src/Form/Form.js [hunk 4]
```diff
@@ -32,15 +33,18 @@ class Form extends React.Component {
...
+  getAllQuestions = async () => {
+    this.props.isLoading(true)
+    await this.props.getQuestions(this.state.round1, 'easy')
+    await this.props.getQuestions(this.state.round2, 'easy')
+    await this.props.getQuestions(this.state.round3, 'medium')
+    await this.props.getQuestions(this.state.round4, 'medium')
+    await this.props.getQuestions(this.state.round5, 'hard')
+    await this.props.getQuestions(this.state.round6, 'hard')
+    this.props.isLoading(false)
   }
```
Comment (KyleWong2510) [inline src/Form/Form.js:48]: Glad the await worked for us to get the arrays in order!

File: src/Form/Form.js [hunk 5]
```diff
@@ -106,9 +110,9 @@ class Form extends React.Component {
 // }
 
 const mapDispatchToProps = dispatch => (
-  bindActionCreators({ setPlayerName, getQuestions }, dispatch)
+  bindActionCreators({ setPlayerName, getQuestions, isLoading }, dispatch)
```
Comment (KyleWong2510) [inline src/Form/Form.js:113]: Still want to know whats happening under the hood with the bindActionCreators...

File: src/Question/Question.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  }
+
+  return (
+    <section className='question-container'>
+      <p>{`${fixedQuestion}`}</p>
+    </section>
+  )
+}
+
+export default Question
```
Comment (KyleWong2510) [inline src/Question/Question.js:20]: nice conditional up top.  that was a tough one!

File: src/App/App.js [hunk 7]
```diff
@@ -6,33 +6,28 @@ import Chalkboard from '../Chalkboard/Chalkboard'
...
+    return (
+      <p>Loading...</p>
+    )
+  }
+
```
Comment (KyleWong2510) [inline src/App/App.js:23]: Good job with the error handling and the isLoading conditional renders!

**Pull request merged**